webapp/src/main/resources/config/application.properties

```diff
@@ -147,6 +147,11 @@ l10n.PluralFormUpdater=true
 #l10n.link.{repository}.commit.url=http://github.com/${gitHubRepository}
 #l10n.link.{repository}.commit.label=${gitHubRepository}
 
+#l10n.link.{repository}.thirdParty.url=http://sometool.com/?
+#l10n.link.{repository}.thirdParty.urlComponent=queryParamToEncode=${assetPath}
```

Comment:
Did you intend to have two equal signs here? (One between `urlComponent` and `queryParamToEncode`, the other between `queryParamToEncode` and `${assetPath}`

Reply:
yes, the first is the delimiter of the property file and after that it can be anything. For a query param `=` is used between the key and the value